src/sionna/phy/fec/ldpc/encoding.py

```diff
@@ -39,23 +40,43 @@ class LDPC5GEncoder(Block):
         Precision used for internal calculations and outputs.
         If set to `None`, :py:attr:`~sionna.phy.config.precision` is used.
 
+    params_only: bool, (default `False`)
+        If set to `True`, the encoder will only initialize the parameters
+        needed for the encoding, but will not load the basegraph or perform
+        any further initialization. This is useful to obtain the parameters
+        of the encoder without initializing the full block.
+
+    allow_low_rates: bool, (default `False`)
+        If set to `True`, the encoder will allow coding rates below 1/3 and 1/5,
+        respectively for BG1 and BG2, consistent with 3GPP specifications.
+
     Input
```

Comment:
Just a hint: the docstrings in the Sionna API are the class docstrings; the docstring of the call() are ignored (only for development). Thus, I've moved the docstrings from the call to here.